src/layer/form-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+# Attributes Form Configuration
+[[toc]]
+
+In addition to setting up [edit widgets](../form-widgets/) and designing the [form layout](../form-layout/), extra configuration can be done to the fields in QGIS to make the data collection easier and more consistent.
+
+
+<YouTube id="jc4p1PpXj3k" />
+
+
+## Default values
+Default values can be used to automatically record, e.g, the name of the surveyor, date and time of the survey, latitude and longitude of the feature, or to have frequently used values filled in advance (see <QGISHelp ver="latest" link="user_manual/working_with_vector/vector_properties.html#default-values" text="QGIS documentation" />).
+
+The default value can be a text, number or a QGIS expression. The data format of the field has to match the result of the default value expression you want to use.
+
+In QGIS, default values can be set up in the **Attributes Form** tab in **Layer Properties**. In the **Defaults** tab, you can enter the default value of the selected field (manually or using an expression). Check the option *Apply default value on update* :heavy_check_mark: to update the field value whenever there is a change in the feature.
+
```

Comment:
Can we please add an info that in case you select the option "Apply default value on update", you won't be able to edit the field in the mobile app?